2024-02-11 16:05:03,953 INFO: Accessed index page [in /srv/talknet/backend/auth-service/app.py:67]
2024-02-11 16:16:53,850 INFO: Accessed index page [in /srv/talknet/backend/auth-service/app.py:99]
2024-02-11 16:17:28,944 INFO: Accessed index page [in /srv/talknet/backend/auth-service/app.py:99]
2024-02-11 16:17:37,710 INFO: Accessed index page [in /srv/talknet/backend/auth-service/app.py:99]
2024-02-11 16:17:37,728 ERROR: Registration error: (psycopg2.OperationalError) connection to server at "localhost" (::1), port 5432 failed: FATAL:  password authentication failed for user "username"
connection to server at "localhost" (::1), port 5432 failed: FATAL:  password authentication failed for user "username"

(Background on this error at: https://sqlalche.me/e/20/e3q8) [in /srv/talknet/backend/auth-service/app.py:121]
2024-02-11 16:17:40,285 INFO: Accessed index page [in /srv/talknet/backend/auth-service/app.py:99]
2024-02-11 16:17:41,228 INFO: Accessed index page [in /srv/talknet/backend/auth-service/app.py:99]
2024-02-11 16:22:46,331 INFO: Accessed index page [in /srv/talknet/backend/auth-service/app.py:99]
2024-02-11 16:23:18,460 ERROR: Registration error: (psycopg2.OperationalError) connection to server at "localhost" (::1), port 5432 failed: FATAL:  password authentication failed for user "username"
connection to server at "localhost" (::1), port 5432 failed: FATAL:  password authentication failed for user "username"

(Background on this error at: https://sqlalche.me/e/20/e3q8) [in /srv/talknet/backend/auth-service/app.py:121]
2024-02-11 16:23:18,530 ERROR: Registration error: (psycopg2.OperationalError) connection to server at "localhost" (::1), port 5432 failed: FATAL:  password authentication failed for user "username"
connection to server at "localhost" (::1), port 5432 failed: FATAL:  password authentication failed for user "username"

(Background on this error at: https://sqlalche.me/e/20/e3q8) [in /srv/talknet/backend/auth-service/app.py:121]
2024-02-11 16:23:18,544 ERROR: Registration error: (psycopg2.OperationalError) connection to server at "localhost" (::1), port 5432 failed: FATAL:  password authentication failed for user "username"
connection to server at "localhost" (::1), port 5432 failed: FATAL:  password authentication failed for user "username"

(Background on this error at: https://sqlalche.me/e/20/e3q8) [in /srv/talknet/backend/auth-service/app.py:121]
2024-02-11 16:23:18,563 ERROR: Registration error: (psycopg2.OperationalError) connection to server at "localhost" (::1), port 5432 failed: FATAL:  password authentication failed for user "username"
connection to server at "localhost" (::1), port 5432 failed: FATAL:  password authentication failed for user "username"

(Background on this error at: https://sqlalche.me/e/20/e3q8) [in /srv/talknet/backend/auth-service/app.py:121]
2024-02-11 16:23:18,630 ERROR: Registration error: (psycopg2.OperationalError) connection to server at "localhost" (::1), port 5432 failed: FATAL:  password authentication failed for user "username"
connection to server at "localhost" (::1), port 5432 failed: FATAL:  password authentication failed for user "username"

(Background on this error at: https://sqlalche.me/e/20/e3q8) [in /srv/talknet/backend/auth-service/app.py:121]
2024-02-11 16:23:18,645 ERROR: Registration error: (psycopg2.OperationalError) connection to server at "localhost" (::1), port 5432 failed: FATAL:  password authentication failed for user "username"
connection to server at "localhost" (::1), port 5432 failed: FATAL:  password authentication failed for user "username"

(Background on this error at: https://sqlalche.me/e/20/e3q8) [in /srv/talknet/backend/auth-service/app.py:121]
2024-02-11 16:23:18,661 ERROR: Registration error: (psycopg2.OperationalError) connection to server at "localhost" (::1), port 5432 failed: FATAL:  password authentication failed for user "username"
connection to server at "localhost" (::1), port 5432 failed: FATAL:  password authentication failed for user "username"

(Background on this error at: https://sqlalche.me/e/20/e3q8) [in /srv/talknet/backend/auth-service/app.py:121]
2024-02-11 16:23:18,926 ERROR: Registration error: (psycopg2.OperationalError) connection to server at "localhost" (::1), port 5432 failed: FATAL:  password authentication failed for user "username"
connection to server at "localhost" (::1), port 5432 failed: FATAL:  password authentication failed for user "username"

(Background on this error at: https://sqlalche.me/e/20/e3q8) [in /srv/talknet/backend/auth-service/app.py:121]
